src/i18n/dictionary.ts

```diff
@@ -1,6 +1,11 @@
 const dictionary = {
 	DEFAULT_BUTTON_ADD: { fr: 'Ajouter une ligne', en: 'Add row' },
 	DEFAULT_BUTTON_REMOVE: { fr: 'Supprimer une ligne', en: 'Remove row' },
+	DEFAULT_BUTTON_REMOVE_THAT_ROW: {
+		fr: 'Supprimer cette ligne',
+		en: 'Remove that row',
```

Comment:
`Delete this row` or `Delete this line` sounds better no?

(At least `that` should be replaced by `this`.)

Reply:
`Delete this row` sounds better since we're working with a table